static/js/src/entry/account/routes/edit-contact-info/Index.vue

```diff
@@ -83,5 +64,24 @@ export default {
       next();
     }
   },
+
+  data() {
+    return {
+      showModal: false,
+      checkModalResolve: () => {},
+    };
+  },
+
+  methods: {
+    setShowModal(shouldShow) {
+      this.showModal = shouldShow;
+    },
+
+    checkModalAction() {
+      return new Promise((resolve) => {
+        this.checkModalResolve = resolve;
+      });
+    },
+  },
```

Comment:
`eslint --fix` just moved this code underneath `beforeRouteLeave`. Nothing about the code itself changed.